docs/getting-started.md

```diff
@@ -80,99 +84,285 @@ await Kilt.init({ address: YOUR_CHAIN_ADDRESS })
 Again, this is asynchronous, so be sure to wrap this in an `async` function as described above.
 Add this line to the `async` functions wrapping the examples below if you run them individually.
 
-## 2. How to generate an Identity
+### 1.2. Connect to a Chain
 
-To generate an Identity first you have to generate a [BIP39 mnemonic](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and then use it to create the Identity:
+Either explicitly:
+
+```typescript
+Kilt.config({ address: YOUR_CHAIN_ADDRESS })
+await Kilt.connect()
+```
+
+Or by setting a default address in the configuration, connecting implicitly.
+
+```typescript
+Kilt.config({ address: YOUR_CHAIN_ADDRESS })
+```
+
+Note that calling (as described in [1.2](#initializing-the-kilt-sdk))
+
+```typescript
+Kilt.init({ address: YOUR_CHAIN_ADDRESS })
+```
+
+initializes the SDK _and_ sets the config, so it is related to the second approach.
+
+There are of KILT chains which you can use, each one having a different address:
+
+1. A local node: `ws://127.0.0.1:9944`
+2. The prod-net: `wss://peregrine.kilt.io`
+3. The dev-net: `wss://kilt-peregrine-stg.kilt.io`
+4. The live-net: `wss://spiritnet.kilt.io`
+
+In case you go with option #1 or #2, you have to request test money **since storing a CTYPE on the chain requires tokens and a full did** as transaction fee and deposit.
+However, **we recommend to start your local node** and use a mnemonic which already has tokens by using our docker image. Depending on which version of the SDK you are using, you might need to spin up the latest officially released version of the node (working with the latest release of the SDK) or the latest development version of the node (working with the latest development version of the SDK). Either version can be started with the following command by using one between the `latest` and `develop` tag.
```

Comment:
I am not sure this is clear for users. It might be very confusing. Not saying we should remove this, but perhaps we could ask someone else's opinion on this?